### PR TITLE
NOMRG - Build wheels for all plateform

### PR DESCRIPTION
**Pull request by Badr-MOUFAD**
This addresses #272 and continues #271 

It will create a workflow to build wheels for
- Linux
- macOS
- Windows

### CODE REVIEW

File: .github/workflows/wheels.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        env:
+          CIBW_BUILD: cp38-manylinux_x86_64 cp38-win32 cp38-macosx_x86_64
+
+      - name: publish to Test PyPI
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.TEST_PYPI_API_TOKEN }}
+        run: |
+          pip install twine
+          twine upload --repository testpypi wheelhouse/
```
Comment (mathurinm) [inline .github/workflows/wheels.yml:43]: if you upload in each job, you may need the `ignore-existing` flag

File: .github/workflows/wheels.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        run: |
+          pip install numpy==1.21
+          pip install -e .
+
+      - name: Install / setup cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==2.8.0
+          python -m cibuildwheel --output-dir wheelhouse
+        env:
+          CIBW_BUILD: cp38-manylinux_x86_64 cp38-win32 cp38-macosx_x86_64
```
[the remark this answers is not in the document]
Comment (Badr-MOUFAD) [reply .github/workflows/wheels.yml:35]: After inspecting the job logs, I don't think so.

For each platform, ``cibuildwheel`` will build wheels for its corresponding params

(cf. https://github.com/mathurinm/celer/runs/7358664075?check_suite_focus=true#step:6:61)

[the remark this answers is not in the document]
Comment (mathurinm) [reply .github/workflows/wheels.yml:35]: Powerful ! So there is no need to run the job on the 3 OS then ?

[the remark this answers is not in the document]
Comment (Badr-MOUFAD) [reply .github/workflows/wheels.yml:35]: The logs suggest the opposite. We should run it on the 3 OS.

File: setup.py [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
 DISTNAME = 'celer'
 DESCRIPTION = descr
-with open('README.md', 'r') as f:
+with open('README.md', 'r', encoding='utf-8') as f:
```
Comment (Badr-MOUFAD) [inline setup.py:20]: @mathurinm,
didn't check that yet, but I trust this will solve the problem of uploading non-ASCII characters
using windows VM

**Comment by mathurinm** [comment]
0.7.2 released !

**Pull request closed without merging**